MLBOM/Model/OpenSource/bom.json

```diff
@@ -0,0 +1,63 @@
+{
+    "$schema": "http://cyclonedx.org/schema/bom-1.6.schema.json",
+    "bomFormat": "CycloneDX",
+    "specVersion": "1.6",
+    "serialNumber": "urn:uuid:21d0b6f8-f5b0-44df-8587-79c5d70cd1da",
+    "version": 1,
+    "metadata": {
+        "timestamp": "2024-11-24T13:10:49Z",
+    },
+    "components": [
+        {
+            "type": "machine-learning-model",
```

Comment:
Same issue here. The type does not have a modelCard property. And there should ideally be a data subcomponent.